ext/FillArraysSparseArraysExt.jl

```diff
@@ -1,10 +1,11 @@
 module FillArraysSparseArraysExt
 
 using SparseArrays
+using SparseArrays: SparseVectorUnion
```

Comment:
This is from one of the ambiguity source https://github.com/JuliaSparse/SparseArrays.jl/blob/a5e95ec23649bd8cb75d2923861c74061016d1dd/src/sparsevector.jl#L1760
so if it changes someday, the ambiguity should also change.

Reply:
Perhaps we should define specific methods for the individual elements from the `Union`, which would not change even if this method signature is altered by `SparseArrays`, or if the type is removed.

Reply:
Seems like it's supposed to be external
https://github.com/JuliaSparse/SparseArrays.jl/blob/a5e95ec23649bd8cb75d2923861c74061016d1dd/src/sparsevector.jl#L92
`# the following aliases are unused internally, but widespread in packages`

Reply:
Can I copy the alias definitions over?

Reply:
I think that's a good idea, and let's define specific methods for each type instead of for the union (perhaps by calling an internal function `_dot` within each method). This is the safest.